cadc-inventory-server/src/test/java/org/opencadc/inventory/transfer/ProtocolsGeneratorTest.java

```diff
@@ -99,10 +100,16 @@ public void testPrioritizePullFromSites() throws Exception {
         for (int i = 0; i < 10; i++) {
             sites.add(new StorageSite(URI.create("ivo://site" + i), "site1" + i, true, rd.nextBoolean()));
         }
-        ProtocolsGenerator.prioritizePullFromSites(sites);
-        for (StorageSite s : sites) {
-            log.info("found: " + s.getID() + " aka " +  s.getResourceID());
-        }
+        List<StorageSite> result1 = ProtocolsGenerator.prioritizePullFromSites(sites);
+        Assert.assertEquals(sites.size(), result1.size());
+        Assert.assertTrue(result1.containsAll(sites));
+        
+        List<StorageSite> result2 = ProtocolsGenerator.prioritizePullFromSites(sites);
```

Comment:
What is this testing?

Reply:
it is just testing that pull from prioritization is random by those two sets not being the same order

Reply:
OK. Does it mean that it is possible to give a false positive when random randomly returns the same order?